tools/gulp/tasks/unit-test.ts

```diff
@@ -1,10 +1,7 @@
 import {join} from 'path';
-import {task, watch} from 'gulp';
+import {task} from 'gulp';
 import {buildConfig, sequenceTask} from '../../package-tools';
 
-// There are no type definitions available for these imports.
-const runSequence = require('run-sequence');
-
 // Default Karma options.
 const defaultOptions = {
```

Comment:
You can inline this since it's only being used in one place now. It should allow us to drop an object spread when creating the server as well.

Reply:
Done.